change/@microsoft-fast-foundation-04b7d5da-6027-4e9f-a908-a416d38194c5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat: calendar attr to adjust weekday start",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "robarb@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
```suggestion
  "dependentChangeType": "prerelease"
```

Reply:
Oops, I half got it set to prerelease. Fixed.

Reply:
yeah it does that automatically, no worries!